mail/common/src/test/java/com/fsck/k9/mail/MimeTypeTest.kt

```diff
@@ -63,12 +69,9 @@ class MimeTypeTest {
     }
 
     private fun assertInvalidMimeType(input: String) {
-        try {
-            input.toMimeType()
-            fail("Expected exception")
-        } catch (e: IllegalArgumentException) {
-            assertThat(e.message).startsWith("Invalid MIME type: ")
-        }
+        assertThat { input.toMimeType() }.isFailure()
+            .isInstanceOf(IllegalArgumentException::class)
+            .message().isNotNull().startsWith("Invalid MIME type: ")
```

Comment:
```suggestion
        assertThat { 
            input.toMimeType() 
        }.isFailure()
            .isInstanceOf(IllegalArgumentException::class)
            .message().isNotNull().startsWith("Invalid MIME type: ")
```

I think writing it like this makes it a bit easier to see at a glance what expression should cause the exception.